theme/lumo/vaadin-message-styles.js

```diff
@@ -8,14 +8,41 @@ registerStyles(
   'vaadin-message',
   css`
     :host {
-      height: calc(var(--lumo-size-l) / 10);
-      margin: var(--lumo-space-s) 0;
+      padding: var(--lumo-space-s);
     }
 
-    [part='value'] {
-      border-radius: var(--lumo-border-radius);
-      background-color: var(--lumo-primary-color);
-      color: white;
+    :host {
+      font-family: var(--lumo-font-family);
+      font-size: var(--lumo-font-size-m);
+    }
+
+    .vaadin-message-wrapper {
+      padding-inline-start: var(--lumo-space-s);
+    }
+
+    /* IE11 is the only browser that we support that doesn't support
+     * padding-inline-start. For IE11, we use padding-left and -right instead. */
+    @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
```

Comment:
Then why even use `padding-inline-start` at all? If we'd use `padding-left` and `padding-right` for everything, we reduce the amount of code, and get rid of this magic `@media` spell, which an average programmer doesn't understand.

Reply:
I think it is a bad practice that we would go with a worse implementation for everything and hide why it is done the way it is done. It feels like we would be creating legacy code from the get-go, due to a browser, which we have not supported in V15 forward. Things like margin-left is why RTL would be very hard to implement in V8, and the lack of them is the reason why it is easier in V10+. Being able to do an out-of-the-box RTL-compatible solution without the problems is IMO a strong thing.

I'm okay going either way, but wanted to share my thoughts on the matter beforehand, but I'm fine with you to decide either one.

I was a bit skeptical on `padding-inline-start` usage myself. @web-padawan do you have an opinion on using it?

Reply:
If it's up to me, let's change it. I understand why `padding-inline-start` is cool, but this IE11 workaround looks so hacky that it defeats the upside. Also, I think using the stable old properties is consistent with other components, based on my memory.

Reply:
We do not use `padding-inline-start` etc anywhere in components at the moment. 
So it might be confusing for any users extending the Lumo theme with their own styles.

I agree with @pekam. IMO it makes sense to use old properties for now.

Reply:
Okay, it is agreed! I'll make the change.

Reply:
Done.